spec/forms/employers_contract_claim_spec.rb

```diff
@@ -19,4 +19,18 @@
     # TODO: Unit tests for upload functionality
 
   end
+
+  describe ".to_h " do
+    it "will take two strings and convert them into a hash" do
+      employers_contract_claim = described_class.new(make_employer_contract_claim: true, claim_information: 'lorem ipsum claim', uploaded_content: 'path/to/file.rtf')
+
+      expect(employers_contract_claim.to_h).to be_a(Hash)
+    end
+
+    it 'will return the value for a key' do
+      employers_contract_claim = described_class.new(make_employer_contract_claim: true)
+
+      expect(employers_contract_claim.to_h).to include(make_employer_contract_claim: true)
+    end
+  end
 end
```

Comment:
All form specs should specify and test all attributes not just 1 like this.  You can prove any type conversions, parsing etc.. then as well

Reply:
Fixed in 1f676c9

Good spot, I forgot that the model won't be doing much once we switch over properly!